lib/abci/handlers/beginBlockHandlerFactory.js

```diff
@@ -33,6 +37,12 @@ function beginBlockHandlerFactory(
   async function beginBlockHandler({ header }) {
     logger.info(`Block begin #${header.height}`);
 
+    const { coreHeight } = header;
+
+    await waitForChainlockedHeight(coreHeight);
+
+    await waitForSMLSync(coreHeight);
```

Comment:
"wait" sounds kinda strange here. Let's rename it to `updateSimplifiedMasternodeList`